test/util/Util.ts

```diff
@@ -43,6 +48,15 @@ export function getPort(name: typeof portNames[number]): number {
   return 6000 + idx;
 }
 
+export function getSocket(name: typeof socketNames[number]): string {
+  const idx = socketNames.indexOf(name);
+  // Just in case something doesn't listen to the typings
+  if (idx < 0) {
+    throw new Error(`Unknown socket name ${name}`);
+  }
+  return `/tmp/css${idx}.sock`;
```

Comment:
When starting a server using this socket, does this actually create anything on the machine at that location or is this just a dummy string that is not really needed? If yes we should put this in the test folder.

Reply:
it creates a socket file at the specified location yes, i will fix it